frontend/test/__support__/ui.tsx

```diff
@@ -99,22 +106,24 @@ function Wrapper({
 function MaybeRouter({
   children,
   hasRouter,
+  initialPath = "/",
 }: {
   children: React.ReactElement;
   hasRouter: boolean;
+  initialPath?: string;
 }): JSX.Element {
   if (!hasRouter) {
     return children;
   }
-  const history = createMemoryHistory({ entries: ["/"] });
+  const history = createMemoryHistory({ entries: [initialPath] });
```

Comment:
Make it possible to render components at other paths. This could be useful in places that rely on location pathname e.g. `getRouterPath` in 
https://github.com/metabase/metabase/blob/17e78d2589ab12ae3d736dc20a79548269d39994/frontend/src/metabase/selectors/app.ts#L52